.github/workflows/pr-testing.yml

```diff
@@ -14,9 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node:
-        - 14
-        - 16
+        node: 16
```

Comment:
Odd that it had two versions?  Is this a try 14 and then fall back to 16 or vice versa?